ers/filejson.go

```diff
@@ -65,7 +65,7 @@ type JSONFileER struct {
 	cgrCfg        *config.CGRConfig
 	cfgIdx        int // index of config instance within ERsCfg.Readers
 	fltrS         *engine.FilterS
-	rdrDir        string
+	dir           string
```

Comment:
Is there a strong reason for renaming?

Reply:
Not really, it just felt repetitive to always use rdr.rdrDir when adding the new ERs functionality.

As per https://google.github.io/styleguide/go/guide#naming:

> Names should:
> 1. Not feel [repetitive](https://google.github.io/styleguide/go/decisions#repetition) when they are used

The case I talked about above can be included here (rdr.rdrDir can be considered repetitive)
> 2. Take the context into consideration:

dir is found inside a CSV/FWV/JSONFileER which represent file event readers (also implementing the EventReader interface), so we can infer that dir is referring to the directory handled by the reader.
> 3. Not repeat concepts that are already clear


This change should be addressing all of the above.